src/app/dataset-block/metadata-block/components/event-details/event-details.component.ts

```diff
@@ -65,6 +68,7 @@ export class EventDetailsComponent implements AfterViewInit, OnChanges {
             );
             componentRef.setInput("event", this.block.event);
         }
+        this.cdr.detectChanges();
```

Comment:
Looks scary: iniating change detection inside a method called from change detection callback.. Why is this needed? Can we live with `markForCheck` instead of forcing?